test/helper.rb

```diff
@@ -23,4 +23,4 @@ def method_missing(method, *args)
 end
 
 require "fluent/plugin/out_map"
-require "fluent/plugin/filter_map" if defined?(Fluent::Filter)
+require "fluent/plugin/filter_map" if defined?(Fluent::Plugin::Filter)
```

Comment:
I don't want to remove out_map right now.
Should we remove out_map in this plugin?